src/packages/tauv_common/src/vision/starter_task/threshold_node.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from sensor_msgs.msg import Image
+import numpy as np
+import cv2 as cv
+import rospy
+from cv_bridge import CvBridge
+from tauv_common.srv import SetThresholdRequest, SetThresholdResponse
+
+LOWER_THRESHOLD = np.array([0, 60, 60]) # TODO: figure out typing for threshold
+UPPER_THRESHOLD = np.array([15, 255, 255])
+
+class ThresholdNode(): 
+    def __init__(self):
+        print("running!")
+        rospy.init_node('starter_threshold')
+
+        self._lower_threshold : np.ndarray = LOWER_THRESHOLD
+        self._upper_threshold : np.ndarray = UPPER_THRESHOLD
+
+        self._cv_bridge : CvBridge = CvBridge()
+        self._publisher : rospy.Publisher = rospy.Publisher('starter_task/thresholded_img', Image, queue_size=5)
+
+        rospy.Subscriber('/kf/vehicle/oakd_bottom/stereo/right/image_color', Image, self.filter)
```

Comment:
Shouldn't need to hardcode `/kf` here. Using `vehicle/...` (no leading `/`) will set the `/kf` automatically using ROS namespacing if the launch file is set up properly.